src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -898,7 +898,7 @@ EnsureSupportedSequenceColumnType(Oid sequenceOid)
 	{
 		ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 						errmsg("cannot create an mx table with columns which use "
-							   "sequences, but are not BIGINT")));
+							   "sequences, but are not bigserial")));
```

Comment:
Yeah but couldn't prepare a better one. I mean it would be nice if we could improve this error message. 

I am not sure but does something like `errmsg("cannot create an mx table with serial columns"),errhint("please use bigserial instead")` make sense to you?